pkgs/applications/misc/calibre/default.nix

```diff
@@ -28,15 +27,59 @@
 , xdg-utils
 , wrapGAppsHook
 , unrarSupport ? false
+, isocodes
+, cacert
+, zip
+, fetchFromGitHub
+, sphinx
+, liberation_ttf
+, mathjax-src ? null
+, hyphenation-src ? null
+, translations-src ? null
 }:
+let
+  nullOrX = x: y: if x == null then y else x;
```

Comment:
Nit: instead of `nullOrX`, how about calling this `setDefaultPackage`? Sounds more descriptive.

Reply:
Now this function is removed.